src/picosvg/svg_types.py

```diff
@@ -174,6 +180,16 @@ def end(self):
     def as_path(self) -> "SVGPath":
         return self
 
+    def remove_overlaps(self, inplace=False, is_clip_path=False) -> "SVGPath":
```

Comment:
nit: suggest passing fill rule; remove overlaps having to know if it's a clip path seems odd to me

Reply:
I will change to pass `fill_rule` as a parameter, but I will keep it optional (default to None), because an SVGPath (self) already knows its `fill_rule` so it was as well just use that.. If only SVG wasn't so complicated and one needs to use `clip_rule` instead of `fill_rule` attribute when performing union/intersection in a clipPath context.

Reply:
fill_rule None seems very odd.